src/directives/text-fit/text-fit.js

```diff
@@ -14,41 +14,62 @@
 //   on the shouldWarn parameter.  This allows the caller to retry again later,
 //   perhaps after the element has become visible.
 angular.module( 'gizmos.directives' ).value( 'textFit', function textFit( element, options, shouldWarn ) {
-  var min, max, mid, lastMid, containerWidth, containerHeight
-
-  element = angular.element( element )
+  var min, max, mid, lastMid, containerStyle, containerWidth, containerHeight, projectedPercentageOfBox, accuracy, allowWordWrap
+  
+  // ToDo: get options from text-fit-group
   options = options || {}
 
-  // Min and max font size.
-  min = options.min || 6
-  max = options.max || 20
+  if ( options.debug ) console.log('[textFit] Running on: ', element.text())
 
-  containerWidth = element.parent().width()
-  containerHeight = element.parent().height()
+  // Check if element is hidden
+  if ( element[0].offsetHeight === 0 ){
+    if ( options.debug ) console.log('[textFit] hidden element: ', element.text())
+    return null
+  }
+  
+  // Set accuracy for faster guessing
+  accuracy = options.accuracy || 0
+  
+  // dis-allow word wrap
+  allowWordWrap = !(options.wordWrap === false)
```

Comment:
I'm confused by this - can't this be reduced to
```javascript
if ( !!options.wordWrap ) {
  element.css('white-space', 'nowrap')
}
```?
I don't see the `allowWordWrap` variable used anywhere else